plugins/fe_events_direct/lang/en/local_direct.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+defined('MOODLE_INTERNAL') || die();
+
+$string['pluginname'] = 'Direct to EventAPI';
+$string['pluginname_help'] = 'Event testing straight to EventAPI';
```

Comment:
This file is named incorrectly. You'll notice these strings don't appear in the menu where you'd expect them to and the file name is the culprit.

Reply:
Would the appropriate file rename here have to be `fe_events_direct.php` ? Or does it need to be shortened?

Reply:
I believe it should be `local_fe_events_direct.php` but you may want to confirm / test that.